crates/edr_napi/index.d.ts

```diff
@@ -418,7 +418,8 @@ export class Provider {
   setVerboseTracing(verboseTracing: boolean): void
 }
 export class Response {
-  get json(): string
+  /** Returns the response data as a JSON string or a JSON object. */
+  get data(): string | any
```

Comment:
In TypeScript, any type in a union with `any` becomes `any` (like multiplying by 0 or, more precisely, doing a set union with the universal set). So this is just `any` and not ideal.

I'm not 100% what the right type would be here. I think `string | object` would work. But if that's not easily doable with N-API, then it's fine for this to be `any` for now.

Reply:
I can look into it but the `serde_json::Value` translates to any